### PR TITLE
Docs - Static and Dynamic Boards concept

### LINKED ISSUE #3933: [Docs] Add concepts of static and dynamic boards to Metrics in Meshery page
#### Current State
Concepts discussed in the Wed. August, 18th, 2021 Meshery Development Meeting, pictorially described in the Meshery Architecture deck, and described in a google doc in the community drive on the subject of static and dynamic boards for the collection of metrics from Prometheus are missing in Meshery docs.

#### Desired State
Publish these resources here - https://docs.meshery.io/guides/meshery-metrics

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by suhaniii**
Signed-off-by: Suhani Agarwal <suhani2612@gmail.com>

**Description**

This PR fixes #3933 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 80090d758addb044005e9526698e02f724b3cf15 at: https://61211629f96e11277ea89416--meshery-docs-preview.netlify.app

File: docs/pages/guides/meshery-metrics.md [hunk 1]
```diff
@@ -14,6 +14,29 @@ Meshery provides performance reports, including performance test results, node r
...
+
+[![Grafana Charts]({{ site.baseurl }}/assets/img/architecture/GrafanaCharts.svg)]({{ site.baseurl }}/assets/img/architecture/GrafanaCharts.svg)
+
+### Dynamic Boards
+
+Dynamic Boards can be generated from prometheus or grafana. These borads are defined by user. Grafana SDK is used for these boards.
+
+### Static Boards
+
+Static Borads capture the Service Mesh Performance. Certain protos are defined and tracked as a part of performace tests. Static Boards Queries prometheus SDK directly.
```
Comment (warunicorn19) [inline docs/pages/guides/meshery-metrics.md:36]:
```suggestion
Static Boards capture the Service Mesh Performance. Certain protos are defined and tracked as a part of performance tests. Static Boards Queries Prometheus SDK directly.
```
consider using this instead

File: docs/pages/guides/meshery-metrics.md [hunk 2]
```diff
@@ -14,6 +14,29 @@ Meshery provides performance reports, including performance test results, node r
...
+
+### Grafana Charts
+
+User needs to set the grafana URL and API key to create and query boards.
+
+[![Grafana Charts]({{ site.baseurl }}/assets/img/architecture/GrafanaCharts.svg)]({{ site.baseurl }}/assets/img/architecture/GrafanaCharts.svg)
+
+### Dynamic Boards
+
+Dynamic Boards can be generated from prometheus or grafana. These borads are defined by user. Grafana SDK is used for these boards.
```
Comment (warunicorn19) [inline docs/pages/guides/meshery-metrics.md:32]:
```suggestion
Dynamic Boards can be generated from Prometheus or Grafana. These boards are defined by the user. Grafana SDK is used for these boards.
```
consider using this instead.

File: docs/pages/guides/meshery-metrics.md [hunk 3]
```diff
@@ -14,6 +14,29 @@ Meshery provides performance reports, including performance test results, node r
 
 In order to pull in these environment metrics, you can also manually configure Meshery to connect with your existing Grafana and/or Prometheus instances through the Meshery dashboard. Once they have been loaded and are displayed on the screen, you may also perform an *ad-hoc* test to check Meshery's connection status.
 
+### Prometheus Charts
+
+User needs to set the prometheus URL and API key to create and query boards.
```
Comment (sayantan1413) [inline docs/pages/guides/meshery-metrics.md:19]:
```suggestion
User needs to set the Prometheus URL and API key to create and query boards.
```

**Review by sayantan1413 — CHANGES_REQUESTED** [review]

**Comment by sayantan1413** [comment]
Screenshot 2021-08-21 at 10 37 39 PM
Screenshot 2021-08-21 at 10 37 29 P@suhaniii rc=
@suhaniii the images look a bit odd here.

**Comment by suhaniii** [comment]
> Screenshot 2021-08-21 at 10 37 39 PM
> 
> Screenshot 2021-08-21 at 10 37 29 PM
> 
> @suhaniii the images look a bit odd here.

@sayantan1413 Right, it does, I took a screenshot, converted it into .svc format online and finally download... Is there a better way?

File: docs/pages/guides/meshery-metrics.md (hunk 1 above)
Comment (suhaniii) [reply docs/pages/guides/meshery-metrics.md]: Thanks, I have added all there changes

**Comment by l5io** [comment]
🚀 Preview for commit 4b9b22b86df32a340e617e6288d09f0e087a2be8 at: https://612167ebf96e11a757a89207--meshery-docs-preview.netlify.app

**Comment by sayantan1413** [comment]
@suhaniii I think we can use PNG as well here if svg is creating a problem.

**Comment by l5io** [comment]
🚀 Preview for commit 2f4f39396563adce8d1ea6fab554df6e929d5263 at: https://61226c170f70c9a913ba7e7e--meshery-docs-preview.netlify.app

**Review by warunicorn19 — APPROVED** [review]
LGTM 🎉! Thanks for this @suhaniii

**Comment by sayantan1413** [comment]
Screenshot 2021-08-23 at 12 07 00 AM
Hey @suhaniii I was thinking that if those 5px solid black border at the bottom and top of those two pictures are removed it would look better I guess. What's your view on this @suhaniii ?😅

**Comment by suhaniii** [comment]
> Screenshot 2021-08-23 at 12 07 00 AM
> 
> Hey @suhaniii I was thinking that if those 5px solid black border at the bottom and top of those two pictures are removed it would look better I guess. What's your view on this @suhaniii ?😅

You are right @sayantan1413

**Comment by l5io** [comment]
🚀 Preview for commit 3b6cf4383d937e585bd4099fce5629a7381b4d67 at: https://61238fec85cd081287cdee5d--meshery-docs-preview.netlify.app

**Review by sayantan1413 — APPROVED** [review]
LGTM @suhaniii 🎉🚀

**Review by leecalcote — CHANGES_REQUESTED** [review]
Please see these instructions - https://github.com/meshery/meshery/issues/3933#issuecomment-902296764

**Comment by l5io** [comment]
🚀 Preview for commit 970e03753a71c1c1c2fcc84af86b9cbe1ee3eeb4 at: https://6125ad63e7f1673f8670c1e1--meshery-docs-preview.netlify.app

**Review by warunicorn19 — APPROVED** [review]

**Review by sayantan1413 — APPROVED** [review]
The svg looks good now ✔️🚀

**Pull request merged**